featurebyte/service/offline_store_feature_table_manager.py

```diff
@@ -695,3 +695,28 @@ async def _update_table_and_column_comments(
         await self.offline_store_feature_table_comment_service.apply_comments(
             feature_store_model, comments, update_progress
         )
+
+    async def update_table_deployment_reference(
+        self, feature_id: ObjectId, deployment_id: ObjectId, to_enable: bool
+    ) -> None:
+        """
+        Update deployment reference in offline store feature tables
+
+        Parameters
+        ----------
+        feature_id: ObjectId
+            Feature to update
+        deployment_id: ObjectId
+            Deployment to update
+        to_enable: bool
+            Whether to enable or disable the deployment
+        """
+        if to_enable:
+            update = {"$addToSet": {"deployment_ids": deployment_id}}
+        else:
+            update = {"$pull": {"deployment_ids": deployment_id}}
+
+        await self.offline_store_feature_table_service.update_documents(
+            query_filter={"feature_ids": feature_id},
+            update=update,
+        )
```

Comment:
Would it make sense to update the references in bulk using a query filter like this:

```
query_filter={"feature_ids": {"$in": feature_ids}}
```

so we don't have to call this per feature?